fvm/transactionInvoker.go

```diff
@@ -180,7 +182,7 @@ func (i *TransactionInvoker) Process(
 				Uint64("blockHeight", blockHeight).
 				Msg("transaction fee deduction executed with error")
 
-			return feesError
```

Comment:
is there a reason to call programs.Cleanup with an empty set there?

Reply:
I've ready cleaned this up as part of #2963